fe/estimator.py

```diff
@@ -25,16 +24,15 @@
 class SimulationResult:
     xs: np.array
     du_dls: np.array
-    du_dps: np.array
 
 
 def flatten(v):
-    return tuple(), (v.xs, v.du_dls, v.du_dps)
+    return tuple(), (v.xs, v.du_dls)
 
 
 def unflatten(aux_data, children):
-    xs, du_dls, du_dps = aux_data
-    return SimulationResult(xs, du_dls, du_dps)
+    xs, du_dls = aux_data
+    return SimulationResult(xs, du_dls)
 
 
 jax.tree_util.register_pytree_node(SimulationResult, flatten, unflatten)
```

Comment:
Tangential to this PR, but this might also now be unused (which would also let us remove `flatten` and `unflatten`)

Reply:
Removed in 3ab3b17